### ISSUE TITLE
Undefined hyper references

### ISSUE THREAD

**Issue by humitos** [cacrespo/tutorial_python_ES#18]
Latex está reportando estos warnings sobre hipervínculos. Sin embargo, no endiendo porqué no están definidos.

```
LaTeX Warning: Hyper reference `tutorial/datastructures:tut-loopidioms' on page
 28 undefined on input line 1015.
```

Entiendo que esto no es un problema, principalmente porque el link funciona bien en el PDF:

![Screenshot_2022-11-05_10-13-16](https://user-images.githubusercontent.com/244656/200112556-3cad80b4-17e3-4635-9829-a6a4adea9353.png)

_(https://github.com/python/cpython/blob/b3cafb60afeb2300002af9982d43703435b8302d/Doc/tutorial/controlflow.rst?plain=1#L135-L136)_

Si no entiendo mal, esto lo muestra Latex en la **primera pasada** cuando todavía las referencias no fueron calculadas. Sin embargo, en la siguiente pasada no se vuelven a mostrar estos warnings.

**Comment by cacrespo** [comment]
Ni me calentaría por los hipervínculos. De hecho, dado que el foco es la versión papel los quitaría todos para reducir margen de error ¿?

**Comment by humitos** [comment]
Sí. En realidad a mí no me molestan los hipervínculos. Lo que quise indicar en este issue es que "puede ser" que haya hipervínculos (referencias) a otros capítulos de la documentación que no **no están en el tutorial** y puede quedar raro de leer. Por ejemplo: "Mira el módulo `toml` en la referencia de módulos"

En ese caso, como no tenemos el módulo `toml` en nuestro tutorial, tendríamos que ver cómo se visualiza en el tutorial. Todavía no encontré ninguno de estos casos, pero lo quería dejar asentado en algún lado.

**Comment by humitos** [comment]
Para estos casos decidimos convertir estas referencias inexistentes en links y poner la URL como nota al pié de página. Hay un ejemplo en #28
